src/lib/db/query.js

```diff
@@ -46,8 +40,9 @@ module.exports = {
       'SELECT': 'date_time::DATE AS date, dataset, count(*)::INTEGER',
       'FROM': 'lev_audit',
       'WHERE': [from && fromDate, to && toDate, group && searchGroup],
-      'GROUP BY': 'date_time::date, dataset ORDER BY date_time::date'
-    }),
+      'GROUP BY': 'date, dataset',
+      'ORDER BY': 'date'
```

Comment:
I wonder if this should be `date_time` rather than `date`.

Reply:
Any reason for the extra space on line 17?

Reply:
I don't think so, `date` is the grouped item. I think using `date_time` would give unpredictable results.

Reply:
The point I was making is that date might prevent the use of the index. But once `GROUP BY` has done its work the data set is probably small enough that it doens't matter.